packages/node_modules/@ciscospark/widget-message-meet/src/components/activity-menu/index.js

```diff
@@ -26,10 +26,12 @@ export default function MessageMeetActivityMenu({
     }
   ];
   return (
-    <ActivityMenu
-      buttons={buttons}
-      onExit={onExit}
-    />
+    <div className={styles.activityMenuContainer}>
```

Comment:
Ok, I'm sure we'll need to go through at some point and make sure everything is user configurable, but that is WAY down the priority list.